ansible_navigator/ui_framework/colorize.py

```diff
@@ -104,7 +107,27 @@ def render(self, doc, scope):
             lines = []
             for line_idx, line in enumerate(doc.splitlines()):
                 first_line = line_idx == 0
-                state, regions = tokenize(compiler, state, line, first_line)
+                try:
+                    state, regions = tokenize(compiler, state, line, first_line)
+                except Exception as exc:  # pylint: disable=broad-except
+                    self._logger.critical(
+                        (
+                            "An unexpected error occured within the tokenization"
+                            " subsystem.  Please log an issue with the following:"
+                        )
+                    )
+                    self._logger.critical(
+                        "  Err: '%s', Scope: '%s', Line follows....", str(exc), scope
+                    )
+                    self._logger.critical("  '%s'", line)
+                    self._logger.critical(
+                        (
+                            "  Please try again, or disable the text-based user"
+                            " interface with '-m stdout' at the command line."
+                        )
+                    )
+                    continue
```

Comment:
What is the actual net result of `continue` here, and skipping the line? Do we not render the line in output at all? Or is it just not colored? At a quick glance it looks like we'd just not render the line at all which is probably nonideal. But I'm not familiar enough with this subsystem to be sure.

Reply:
I was thinking the same thing, we should return the line as is. Will revise.... Good catch